release/_bom.sh

```diff
@@ -213,6 +213,18 @@ function generate_pom_release_dxp_bom_compile_only {
 	) >> "${pom_file_name}"
 }
 
+function generate_pom_release_dxp_distro {
```

Comment:
@pgydev, please sort the methods following alphabetic order. This method should be placed after the method `generate_pom_release_dxp_bom_third_party`.